src/GeositeFramework/js/BasemapSelector.js

```diff
@@ -15,6 +15,9 @@
                hideBasemapList(view, e);
            }
         });
+        $('#help-overlay-start').on('click', function(e) {
+            hideBasemapList(view, e);
```

Comment:
I'm curious why we need to create a listener just for this element? I thought it would be captured by the listener you created in the previous commit that is listening to clicks on `document`. My thought is that if that listener isn't capturing all the clicks, we may have to add more of these special case listeners.